### ISSUE TITLE
Cannot get playgame.py to work

### ISSUE THREAD

**Issue by rhadoo32** [aichallenge/aichallenge#121]
Hi, I am getting this error when running the script playgame,py. Specifying -R or not, makes no difference.

Creating canvas...
Parsing the replay...
Parsing the replay...

Error message:
replaydata.revision should be 2, but was found to be undefined!

Error fileName:
file:///home/radu/workspace/aichallenge/ants/visualizer/js/Replay.js

Error lineNumber:
172

Error stack:
([object Object],"revision",2)@file:///home/radu/workspace/aichallenge/ants/visualizer/js/Replay.js:172
Replay([object Object])@file:///home/radu/workspace/aichallenge/ants/visualizer/js/Replay.js:226
()@file:///home/radu/workspace/aichallenge/ants/visualizer/js/Application.js:376
(0)@file:///home/radu/workspace/aichallenge/ants/visualizer/js/Application.js:256

Error name:
Error

Also, in terminal, Python run gives this log:
Traceback (most recent call last):
  File "../worker/engine.py", line 42, in run_game
    sandbox.start(bot_cmd)
  File "../worker/sandbox.py", line 175, in start
    cwd=working_directory)
  File "/usr/lib/python2.7/subprocess.py", line 672, in **init**
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1213, in _execute_child
    raise child_exception
OSError: [Errno 13] Permission denied

**Comment by amstan** [comment]
Seems like it's trying to use the sandbox, hmm.

**Comment by rhadoo32** [comment]
I don't know....I will try more when I get back from work, but, I really found the install.md hard to follow (but I'm not a Linux expert).
And, after install.md, there is no getting started guide, there is just some code comment in Python.
So, I may have done some wrong...

**Comment by McLeopold** [comment]
The visualizer will not work because a game is not actually being played.  Pass the "--nolaunch" flag to playgame until this works so it doesn't get in your way.

The engine can't start one of the bots correctly.  Which OS?  Are you running play_one_game.*?  Are you trying to run a custom bot?  Can you give us a list of file permissions for the bots that are being run?

**Comment by rhadoo32** [comment]
Sorry, I cannot replicate behavior after reinstall....for some reason. Closing..
OS: Ubuntu 11.04, script is playgame.py, bots are the ones provided here.

**Comment by amstan** [comment]
Just a note.. Install.md is intended to install the contest framework(as in.. start your own contest website), it's not intended for testing your bot in games, you don't need install for that.

**Comment by McLeopold** [comment]
but we do need a tools download...
